var/spack/repos/builtin/packages/py-scitools-iris/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyScitoolsIris(PythonPackage):
+    """A powerful, format-agnostic, community-driven Python library for
+    analysing and visualising Earth science data"""
+
+    homepage = "http://scitools.org.uk"
+    url      = "https://pypi.io/packages/source/s/scitools-iris/scitools-iris-2.2.0.tar.gz"
+
+    version('2.2.0', sha256='1bf8853f5d7a210f711636d32a52ff62b84a56330fe159720ef56f36f3804ade')
+
+    patch('remove_site_cfg.patch', when='@2.2.0')
```

Comment:
Will this patch be needed on other versions?  Can you explain (in comments) briefly why it's needed at all?